tests/Kernel/Paragraphs/ParagraphsTest.php

```diff
@@ -194,64 +188,45 @@ public function testListItem(): void {
     $html = $this->renderParagraph($paragraph);
     $crawler = new Crawler($html);
 
-    $this->assertCount(1, $crawler->filter('.ecl-list-item.ecl-list-item--highlight'));
-    $this->assertEquals('Item title', trim($crawler->filter('.ecl-list-item__title')->text()));
-    // The description should not be rendered in this variant.
-    $this->assertCount(0, $crawler->filter('.ecl-list-item__detail'));
-    // Neither the date.
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__week-day'));
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__day'));
-    $this->assertCount(0, $crawler->filter('.ecl-date-block__month'));
-    // Neither the metas.
-    $this->assertCount(0, $crawler->filter('.ecl-meta__item'));
```

Comment:
We need to keep all the negative assertion above, this will make sure those areas are not filled on the card when we use it for an highlight variant:

![image](https://user-images.githubusercontent.com/153362/62760923-89649d80-ba85-11e9-9fd7-bb14a0233923.png)

Reply:
`ecl-card__description` and `ecl-card__meta` are added.